bittensor/commands/utils.py

```diff
@@ -186,10 +186,10 @@ def filter_netuids_by_registered_hotkeys(
         )
         netuids_with_registered_hotkeys.extend(netuids_list)
 
-    if cli.config.netuids == None or cli.config.netuids == []:
+    if cli.config.netuids is None:
```

Comment:
Are we sure we don't want to do `if not cli.config.netuids:`?

Reply:
Good one 👍 incorporated